src/main/java/sirius/web/dispatch/HelpDispatcher.java

```diff
@@ -38,14 +35,29 @@ public boolean preDispatch(WebContext ctx) throws Exception {
     }
 
     @Override
+    public int getPriority() {
+        return PriorityCollector.DEFAULT_PRIORITY + 100;
+    }
+
+    @Override
     public boolean dispatch(WebContext ctx) throws Exception {
         if (!ctx.getRequest().uri().startsWith("/help") || HttpMethod.GET != ctx.getRequest().method()) {
             return false;
         }
         String uri = ctx.getRequestedURI();
+        String lang = "";
```

Comment:
Should be set to a default Lang so when i call '/help/' the lang parameter given to the rhythm template is the default lang and not an empty String